### DISCUSSION TITLE
Weasis displays two different patient names for one patient. Where does the other one come from please?

### DISCUSSION THREAD

**Discussion by jonathancolledge** [nroduit/Weasis#550] [Q&A] [answered]
![Weasis_Name_1](https://github.com/nroduit/Weasis/assets/4342400/a5e07cf2-9905-4f6b-a7dd-aa5ffce32f8b)

![Weasis_Name_2](https://github.com/nroduit/Weasis/assets/4342400/c571944d-17c9-47d0-abc0-1adf7ae9bf15)

I have an anonymised file, but it has been anonymised with two names and only Weasis can see the name Sergius Cornelius. I cannot find that name in the metadata/tags anywhere. Where is it coming from? Where does Weasis pull the name from when showing it in the folder tab or drop down (I can see it pull the image information for the overlay from 0010,0010). There are also two patient ids,but I can't find the other either. Can anyone help please?

**Comment by nroduit** [comment, marked as answer]
Sounds like a problem of inconsistent metadata within a series or exam. You should find a WARN [log](https://weasis.org/en/tutorials/logging/) describing the problem, or alternatively use the GUI with the [DICOM information part (choose mode A)](https://weasis.org/en/tutorials/tags/) and scroll through the series to see if the value is still the same.

I believe that when there are inconsistencies, the last value is taken according to the reading order.

**Reply by jonathancolledge** [reply]
Thank you very much! It was a single DICOM file that was the result of a first aborted run of the anonymiser. It could be seen as you suggested, using the DICOM information layout and scrolling through to watch it change.
